examples/node-nx-express/.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Why were all the Nx files edited in this PR?

Reply:
The NX tests (`node-nx-express`, `node-nx-next`, etc..) were all using the same path (`./examples/node-nx`), so I separated them up to be more verbose.